stingray/events.py

```diff
@@ -164,6 +166,15 @@ def __init__(self, time=None, energy=None, ncounts=None, mjdref=0, dt=0,
             if self.time.size != self.energy.size:
                 raise ValueError('Lengths of time and energy must be equal.')
 
+    def array_attrs(self):
```

Comment:
What does this function do? Return a list of the names of all the array attributes? Of their values?